packages/flipper_login/lib/login_view.dart

```diff
@@ -0,0 +1,45 @@
+library flipper_login;
+
+import 'package:flutter/material.dart';
+import 'package:universal_platform/universal_platform.dart';
+
+class LoginView extends StatefulWidget {
+  const LoginView({Key? key}) : super(key: key);
+
+  @override
+  LoginViewState createState() => LoginViewState();
+}
+
+class LoginViewState extends State<LoginView> {
+  @override
+  Widget build(BuildContext context) {
```

Comment:
I like the use of the library to know the platform which we are on but it is misused for this case, it should be used for a case where we want to decide functionality that is on one platform not other, so basically for eg: all app should share the same code i.e one Login but differ to small details.

Reply:
so we should not have 4 different UI

Reply:
The PR is still in-progress and need the suggestions

Reply:
To put it into simple explanation we should have one LoginView (Widget) that is responsive to all platform we have given to different screen size you can learn more about responsiveness  [Here](https://github.com/yegobox/flutter_facebook_responsive_ui)

Reply:
Got it